code/game/verbs/rolldice.dm

```diff
@@ -0,0 +1,11 @@
+/client/verb/rolldice()
+	set name = "Roll the Dice!"
+	set desc = "Rolls the Dice of your choice!"
+	set category = "OOC"
+
+	var/list/choice = list(10, 20, 50, 100)
+	var/input = input("Select the Dice you want!", "Dice", null, null) in choice
+	var/conclusion = "[usr] rolls [rand(1,input)] out of [input]!"
+
+	for(var/mob/O in viewers(src))

```

Comment:
Can't this just use a visible_message proc or something?

Reply:
I used a to_chat instead.
I'd say I'd be doing it for the sake of following after the OOC model when it comes to verbs and hence that instead of visible_message; I'm trying to see if it'd be viable to somehow find a way to mute a player from using the verb, versus staff just using the powers they have now to deter someone from doing that.

Reply:
Alternatively, we could just change it so that it only rolls for the player, and they could choose to announce it themselves over LOOC.